### DISCUSSION TITLE
Function Switch configuration on model setup page is missing - Boxer running V2.10.1

### DISCUSSION THREAD

**Discussion by neilyp** [EdgeTX/edgetx#5186] [Q&A]
In the V2.10.1 manual for monochrome screens, a section is shown called function switches which allows you to change the behavior of the 6 function buttons above the display - allowing you to break them up into smaller groups, or use them individually. On my Boxer running V2.10.1 this option is missing. It is also missing in the companion software. It doesn't appear to be in earlier firmware versions either. Is there something I have to do to enable this feature, or has it been removed?

**Comment by inventor7777** [comment]
Check under the "Hardware" category instead. I think that's where you configure switches/hardware.

**Comment by pfeerick** [comment]
It is missing because it is a feature not supported by that radio hardware. Presently, only Jumper radios have this capability - and even then, specifically only the TPro/TProV2, T20/T20S, and T15. On all other radios with six position "flight mode" buttons, it is an analog 6POS input, and we have no control over what led is lit, and is not compatible with the customisable switches functionality

**Reply by neilyp** [reply]
Hi guys and thanks for your reply. Ya I figured it might be hardware related, and thus there is no coding your way out of it. Also I have to remember that the manual is not just for "my" radio.  Oh well, I'll find a use for them somehow.
Thanks again!
